### PR TITLE
Restaurant emission comparison

### PR DESCRIPTION
**Pull request by Robomarti**

### CODE REVIEW

**Comment by Robomarti** [comment]
Still not sure about how safe the localstorage is, since couldn't a user just write whatever restaurantId to localstorage

File: src/lang/en.json [hunk 1]
```diff
@@ -144,6 +144,8 @@
         "MEAL_UPDATED": "Meal updated succesfully!",
         "MEAL_NOT_UPDATED": "Error occured, meal not created.",
         "DOWNLOAD": "Download",
+        "UNAUTHORIZED": "You are not authorized to view this content",
```
Comment (PyryL) [inline src/lang/en.json:147]: Are these translations used somewhere?

**Review by PyryL — APPROVED** [review]
seems ok

File: backend/utils/restaurant.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * Fetch all meal emissions from database.
+ * @param {number} restaurantId
+ * @returns {Promise<{restaurant_name: string}>|[]}
+*/
+const getRestaurantName = async (restaurantId) => {
+    const result = await sql`
+        SELECT name AS restaurant_name
+        FROM restaurants
+        WHERE restaurant_id = ${restaurantId}
+        LIMIT 1;
```
[the remark this answers is not in the document]
Comment (Robomarti) [reply backend/utils/restaurant.js:13]: probably not, since research data might include restaurant name some day

**Pull request merged**